src/runConfigs.ts

```diff
@@ -102,6 +102,11 @@ export interface IExecutionConfig {
          */
         tags?: string[]
     }
+
+    /**
+     * The when clause for matching tests by to determine if the configuration should be run with.
```

Comment:
This can be a new feature in next version, let's add `@since 0.41.0`

Reply:
https://github.com/microsoft/vscode-java-test/pull/1638/commits/338c10dba870cf61cc170143f7b6008d8a0c41b9

Reply:
https://github.com/microsoft/vscode-java-test/pull/1638/commits/751f3d5ab45ebdf17a88712ab3277175b8ee100c

Reply:
https://github.com/microsoft/vscode-java-test/pull/1638/commits/751f3d5ab45ebdf17a88712ab3277175b8ee100c